public/app/themes/clarity/inc/updates.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace MOJ\Justice;
+
+/**
+ * Functions to related to updates, and disabling checks.
+ */
+class Updates
+{
+    public function __construct()
+    {
+        $this->hooks();
+    }
+
+    /**
+     * @return void
+     */
+    public function hooks(): void
+    {
+        /**
+         * Prevent http requests to api.wordpress.org to check for WP core versions.
+         * 
+         * Even though `AUTOMATIC_UPDATER_DISABLED` is set to true, we need to block
+         * requests api.wordpress.org to where WP checks for available versions.
+         * 
+         * This filter's function returns a dummy payload that will prevent an api request, 
+         * and not cause an error.
+         * 
+         * @see https://wp-kama.com/2020/disable-wp-updates-check
+         * @see https://developer.wordpress.org/reference/functions/wp_version_check/
+         */
+
+        add_filter(
+            'pre_site_transient_update_core',
+            fn() =>  (object) [
+                'updates' => [],
+                'version_checked' => $GLOBALS['wp_version'],
+                'last_checked' => time()
+            ]
+        );
+
+        /**
+         * Prevent http requests to api.wordpress.org to check for theme version.
+         * 
+         * This filter's function returns a dummy payload that will prevent an api request,
+         * and not cause an error.
+         * 
+         * @see https://wp-kama.com/2020/disable-wp-updates-check
+         * @see https://developer.wordpress.org/reference/functions/wp_update_themes/
+         */
+
```

Comment:
Hey buddy, I was meant to mention this a while ago. My IDE prefers that the comment sits directly above the method, without a space in between. Are you happy (not now) in future to associate the comment directly with the function/method?

Reply:
Hey, absolutely, that's all good with me.